_examples/pkg/rules/rules.go

```diff
@@ -25,12 +25,23 @@ var (
 	}
 )
 
-func SetupRules() {
+func SetupRules(alertConfigFiles []string) {
```

Comment:
does it mean we need to update the operator's monitoring code?

Reply:
not at all, this is just an example of the possibilities
nothing changes if implementer doesn't want